Jenkinsfile

```diff
@@ -961,7 +965,7 @@ pipeline {
                 Navi21_build_cmd = "LLVM_PATH=/opt/rocm/llvm CTEST_PARALLEL_LEVEL=2 MIOPEN_CONV_PRECISE_ROCBLAS_TIMING=0 MIOPEN_LOG_LEVEL=5 make -j\$(nproc) check"
             }
             parallel{
-                stage('dbsync gfx908') {
+                stage('Dbsync gfx908') {
```

Comment:
[R] We need to add "Dbsync" to the list of available TestSets at line 417. We should have done this a while ago, in PR #2407